components/html-block/html-block.js

```diff
@@ -118,10 +118,15 @@ export const htmlBlockContentStyles = css`
 
 let renderers;
 
-const getRenderers = () => {
+const getRenderers = async() => {
 	if (renderers) return renderers;
-	const tempRenderers = requestInstance(document, 'html-block-renderers');
-	const defaultRenderers = [ new HtmlBlockMathRenderer(), new HtmlBlockCodeRenderer() ];
+	let tempRenderers = requestInstance(document, 'html-block-renderers');
+	if (!tempRenderers) {
+		const tempRendererLoader = requestInstance(document, 'html-block-renderer-loader');
```

Comment:
Nit: this could be named this variable `rendererLoader`.

Reply:
Oh yeah, good point. Just slapped `loader` at the end but I guess it's not really `temp` anymore. 😆

Reply:
Nit: maybe the loader's method should be `getRenderers()` instead of `renderers()`?

Reply:
Yeah, I think that's a better idea. I originally had it implemented as a local getter, which is where the name came from, but that had some quirks I didn't like.